src/components/chat/comment-textbox.tsx

```diff
@@ -40,29 +49,50 @@ export const CommentTextBox: React.FC<IProps> = ({ activeNavTab, numPostedCommen
   };
 
   const handlePostComment = () => {
-    onPostComment?.(commentText);
-    setCommentTextAreaHeight(minTextAreaHeight);
-    setCommentAdded(false);
-    setCommentText("");
-  };
-
-  const handleEscKey = (event: React.KeyboardEvent) => {
-    if(event.key === "Escape") {
+    // do not send post if text area is empty, only has spaces or new lines
+    const content = commentText.trim().replace(/\s\s+/g, " ");
+    if (!(content === "" || content === "\n" || content === " ")) {
+      onPostComment?.(commentText);
       setCommentTextAreaHeight(minTextAreaHeight);
       setCommentAdded(false);
       setCommentText("");
     }
   };
 
+  const handleCommentTextboxKeyDown = (event: React.KeyboardEvent) => {
+    const content = commentText.trim().replace(/\s\s+/g, " ");
```

Comment:
Given that this trim/regex combo as well as the empty test occur in multiple places, how about a utility function like:
```typescript
  function trimContent(content: string) {
    const trimmed = commentText.trim().replace(/\s+\n/g, "\n");
    isEmpty = !trimmed || (trimmed === "\n") || (trimmed === " ");
    return [trimmed, isEmpty] as const;
  }
```